domain/src/main/java/com/aconno/sensorics/domain/repository/FormatRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.aconno.sensorics.domain.repository
+
+import com.aconno.sensorics.domain.format.AdvertisementFormat
+
+interface FormatRepository {
```

Comment:
Rename this, so it indicates that it is a local repository

Reply:
Done.